internal/docs/spec/prop-string.js

```diff
@@ -94,12 +94,12 @@ const getPropString = propData => {
       Currently supports only number and string
     */
     if (propData[name].type.name === 'union') {
-      if (propData[name].value === 'number') {
-        propString += ` ${name}={${propData[name].value}}`
-        return true
-      } else if (typeof propData[name].value === 'string' && propData[name].value.length > 0) {
+      if (typeof propData[name].value === 'string') {
         propString += ` ${name}="${propData[name].value}"`
         return true
+      } else {
```

Comment:
It's risky to have an else case here because we don't really know all the things that come in it.

We can handle the ones we know and leave space for the future case

(maybe you wanted `type.name === 'number'` here)